src/Entities/Player.h

```diff
@@ -499,6 +499,13 @@ class cPlayer :
 	The blocks in range (a_BlockX - a_Range, a_BlockX + a_Range) are sent (NY-metric). */
 	void SendBlocksAround(int a_BlockX, int a_BlockY, int a_BlockZ, int a_Range = 1);
 
+	bool HasSkinPart(eSkinPart a_Part) const { return (m_SkinParts & a_Part) != 0; }
+	UInt8 GetSkinParts(void) const { return m_SkinParts; }
+	void SetSkinParts(UInt8 a_Parts);
```

Comment:
Cannot use `UInt8` for bindings (missing range-checks), use a regular `int` or `unsigned` instead.